packages/reactive-element/src/reactive-element.ts

```diff
@@ -1161,7 +1242,11 @@ export abstract class ReactiveElement
         // Note, it's important that every change has a chance to add the
         // property to `_reflectingProperties`. This ensures setting
         // attribute + property reflects correctly.
-        if (options.reflect === true && this.__reflectingProperty !== name) {
+        if (
+          !initial &&
```

Comment:
Note also that we are not matching platform behavior here precisely wrt reflecting properties. For example, any set to `input.type` will reflect, including not changing the property from its default. 

The code here does not match that since `requestUpdate` immediately returns if there is no value change.

Reply:
removed initial

Reply:
removed `initial`